.github/workflows-src/steps.lib.yml

```diff
@@ -102,7 +102,11 @@ name: "Critical Integration Tests"
 shell: bash
 run: |
   export SHELL=''
+  set +e
   go test -timeout 10m github.com/ActiveState/cli/test/integration -test.v -test.run "^TestLanguageMsiActivePerl|TestDeployIntegrationTestSuite|TestInitIntegrationTestSuite|TestPushIntegrationTestSuite|TestUpdateIntegrationTestSuite|PrepareIntegrationTestSuite$"
+  testCode=$?
+  head -n-0 build/*log 2>/dev/null
```

Comment:
Curious, why `head` and not `cat`?

Reply:
Head prints out the file name before each file body.